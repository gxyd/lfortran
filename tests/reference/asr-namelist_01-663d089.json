{
    "basename": "asr-namelist_01-663d089",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/namelist_01.f90",
    "infile_hash": "8d137496523f2edfdf23149ece78c98a6ae0ce24476ffafe238bb7f6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-namelist_01-663d089.stdout",
    "stdout_hash": "29a3c5a914b2c2e8c61cceca5c485faf60eadc434f922bac760b7773",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}