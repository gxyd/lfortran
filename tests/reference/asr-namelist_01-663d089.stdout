(TranslationUnit
    (SymbolTable
        1
        {
            namelist_01:
                (Program
                    (SymbolTable
                        2
                        {
                            config:
                                (Namelist
                                    2
                                    config
                                    [2 temperature
                                    2 max_iterations
                                    2 model_name]
                                    Public
                                ),
                            max_iterations:
                                (Variable
                                    2
                                    max_iterations
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                    .false.
                                    .false.
                                    ()
                                ),
                            model_name:
                                (Variable
                                    2
                                    model_name
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (String 1 (IntegerConstant 20 (Integer 4) Decimal) ExpressionLength PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                    .false.
                                    .false.
                                    ()
                                ),
                            temperature:
                                (Variable
                                    2
                                    temperature
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                    .false.
                                    .false.
                                    ()
                                )
                        })
                    namelist_01
                    []
                    []
                )
        })
    []
)
